conda_lock/conda_lock.py

```diff
@@ -357,6 +312,7 @@ def make_lock_files(
             platform_overrides=platform_overrides,
             virtual_package_repo=virtual_package_repo,
             required_categories=required_categories if filter_categories else None,
+            pip_support=PIP_SUPPORT,
```

Comment:
Hmm, is this based on before or after #340?

Reply:
This is based on before. I'm going to wait for #340 to me merged first before fixing this PR. Plus, I will apply some of the comments from #316 to this PR. Thats why its [WIP] for now.